Gemfile

```diff
@@ -11,6 +11,7 @@ group :test do
   gem 'metadata-json-lint', '~> 0.0'
   gem 'rspec-puppet-facts', '~> 1.3'
   gem 'semantic_puppet', '0.1.3'
+  gem 'puppet-blacksmith', '>= 3.4.0',      :require => false, :platforms => 'ruby'
```

Comment:
Ugh, yup. I'll re-open this without the blacksmith change - we can get it on to master after. This was sort of a preemptive change.